view/theme/frio/templates/settings_features.tpl

```diff
@@ -19,7 +19,11 @@
 				<div id="{{$g}}-settings-content" class="panel-collapse collapse" role="tabpanel" aria-labelledby="{{$g}}-settings-title">
 					<div class="section-content-tools-wrapper">
 						{{foreach $f.1 as $fcat}}
+							{{* Here is an addional condition to filter the richtext feature
+							because frio doesn't support the richtext *}}
+							{{if $fcat.0 != 'feature_richtext'}}
```

Comment:
> but the run_xgettext script doesn't extract them yet.

would adding a call in theme.php in the theme_init function to `t('some message')` work so that the string is catched by the run_xgettext script? Or would you rather fix the script @fabrixxm ?